cartridges/app_custom_slots/cartridge/controllers/Account.js

```diff
@@ -0,0 +1,182 @@
+
+                    // attempt to create a new user and log that user in.
+                    try {
+                        Transaction.wrap(function () {
+                            var error = {};
+                            var newCustomer = CustomerMgr.createCustomer(login, password);
+
+                            var authenticateCustomerResult = CustomerMgr.authenticateCustomer(login, password);
+                            if (authenticateCustomerResult.status !== 'AUTH_OK') {
+                                error = { authError: true, status: authenticateCustomerResult.status };
+                                throw error;
+                            }
+
+                            authenticatedCustomer = CustomerMgr.loginCustomer(authenticateCustomerResult, false);
+
+                            if (!authenticatedCustomer) {
+                                error = { authError: true, status: authenticateCustomerResult.status };
+                                throw error;
+                            } else {
+                                // assign values to the profile
+                                var newCustomerProfile = newCustomer.getProfile();
+
+                                newCustomerProfile.firstName = registrationForm.firstName;
+                                newCustomerProfile.lastName = registrationForm.lastName;
+                                newCustomerProfile.phoneHome = registrationForm.phone;
+                                newCustomerProfile.email = registrationForm.email;
+                            }
+                        });
+                    } catch (e) {
+                        if (e.authError) {
+                            serverError = true;
+                        } else {
+                            registrationForm.validForm = false;
+                            registrationForm.form.customer.email.valid = false;
+                            registrationForm.form.customer.emailconfirm.valid = false;
+                            registrationForm.form.customer.email.error =
+                                Resource.msg('error.message.username.invalid', 'forms', null);
+                        }
+                    }
+                }
+
+                delete registrationForm.password;
+                delete registrationForm.passwordConfirm;
+                formErrors.removeFormValues(registrationForm.form);
+
+                if (serverError) {
+                    res.setStatusCode(500);
+                    res.json({
+                        success: false,
+                        errorMessage: Resource.msg('error.message.unable.to.create.account', 'login', null)
+                    });
+
+                    return;
+                }
+
+                if (registrationForm.validForm) {
+                    // send a registration email
+                    accountHelpers.sendCreateAccountEmail(authenticatedCustomer.profile, products.getExclusiveProducts());
```

Comment:
You can update accountHelpers.sendCreatedAccountEmail instead of replacing the Account controller.